src/NLog.StructuredLogging.Json.Tests/EndToEnd/ViaLayout/WithFailingLayout.cs

```diff
@@ -57,9 +59,10 @@ public void ShouldSurviveWhenLayoutFails()
         }
 
         [Test]
-        [Category("NotInNetCore")]
         public void ShouldLogFailureWhenLayoutFails()
         {
+            InternalLoggingOn();
```

Comment:
Should we wrap the "off" calls with a ```try-finally``` so that if a test fails, we've not left a global in an inconsistent state for other tests?